curriculum/challenges/english/25-front-end-development/lab-longest-word-in-a-string/a26cbbe9ad8655a977e1ceb5.md

```diff
@@ -0,0 +1,102 @@
+---
+id: a26cbbe9ad8655a977e1ceb5
+title: Build a Longest Word Finder App
+challengeType: 26
+dashedName: build-a-longest-word-finder-app
+---
+
+# --description--
+
+In this lab, you will build a function that returns the length of the longest word in the provided sentence.
+
+For example, in the sentence `"The quick brown fox jumped over the lazy dog"`, the longest word is `"jumped"`, which has a length of 6.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should create a function named `findLongestWordLength` that takes a string as an argument.
+2. The function should return the length of the longest word in the string.
+
+# --hints--
+
+You should create a function named `findLongestWordLength`.
+
+```js
+assert.isFunction(findLongestWordLength);
+```
+
+`findLongestWordLength` should take a string as an argument.
+
+```js
+assert.match(findLongestWordLength.toString(), /\s*function\s+findLongestWordLength\s*\(\s*\w+\s*\)/);
+```
```

Comment:
this test is testing that the function has one parameter, not if it takes a string as an argument